### PR TITLE
Merge subsamples in shape maker, renaming categories and more

### PR DESCRIPTION
**Pull request by andreypz**
A few updates are pending in this PR:
- More advanced handling of subsamples in `convertToRoot` script (the shape maker). The histograms of subsamples are summed.
Other changes include
- Split categories into `mu` and `ele` channels
- Renaming of some categories (to be consistent later with combine naming schemes)
- Introduce functions in CommonSelections that return Cut objects. This will allow to use parameter inside of the cuts directly. Before I had to define separate Cut() objects for each change in the parameter.

### CODE REVIEW

File: CommonSelectors.py [hunk 1]
```diff
@@ -389,83 +396,45 @@ def TrueJetFlavors(events, params, **kwargs):
...
-)
+def ll_antiZ_4j(lep_flav='both'):
+    return  Cut(
+        name = 'll_antiZ_4j',
+        function=AntiZFourJets,
+        params={"lep_flav": lep_flav,
+                "pt_dilep": 60,
+                "mll": {'low': 75, 'high': 120}
+                }
+    )
```
Comment (ValVau) [inline CommonSelectors.py:424]: Looks fine to me. Are you going to transfer 1L and 0L channels as well?

File: workflow_VHcc.py [hunk 2]
```diff
@@ -144,8 +144,8 @@ def count_objects(self, variation):
 
 
     def evaluateBDT(self, data):
-        print(self.params.Models.BDT[self.channel][self.events.metadata["year"]].model_file)
-        print()
+        #print(self.params.Models.BDT[self.channel][self.events.metadata["year"]].model_file)
```
Comment (ValVau) [inline workflow_VHcc.py:147]: In principle, this can be deleted, not just commented out, if you see it work

File: CommonSelectors.py (hunk 1 above)
Comment (andreypz) [reply CommonSelectors.py:424]: Yes, for 1L

File: workflow_VHcc.py (hunk 2 above)
Comment (andreypz) [reply workflow_VHcc.py:147]: It can stay, as it may be useful for debugging later

**Pull request merged**